database/seed.js

```diff
@@ -0,0 +1,82 @@
+const faker = require('faker');
+const db = require('./index.js');
+const Promise = require('bluebird');
+
+const seedRewards = () => {
+  const rewards = [];
+  let tiers;
+  for (let i = 1; i < 101; i++) {
+
+    if (i % 10 === 0 && i % 8 !== 0) {
+      tiers = [1, 5, 10, 25, 50, 75, 100, 250, 500, 750];
+    } else if (i % 8 === 0) {
```

Comment:
If you put this `else if (i % 8 === 0) {` first, you don't need the `i % 8 !== 0` in that first "if"

Reply:
@jordanstholmes Will definitely make that change. Thanks!